http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ResponseHandler.java

```diff
@@ -134,20 +196,18 @@ public void onNext(HttpContent httpContent) {
 
                 @Override
                 public void onError(Throwable t) {
-                    subscriber.onError(t);
+                    runAndLogError("Subsriber#onError threw an exception", () -> subscriber.onError(t));
```

Comment:
Subsriber -> Subscriber (here and elsewhere). Will the subscriber be the customer's? Could it be beneficial to log which one?

Reply:
Yes in this context it will be the customers subscriber. Were you thinking like a class name or something?

Reply:
Yeah, a class name or just calling toString on it. We could implement useful toStrings for our internal implementations?

Reply:
+1 for toString, I'll do that.

Reply:
Added.